tsl/test/sql/compressed_collation.sql

```diff
@@ -13,6 +13,10 @@ select * from (
 ) c
 order by priority limit 1 \gset
 
+\echo priority = :priority
```

Comment:
I assume this is just for testing and will be removed in the final PR?

Reply:
It will... I'm debugging it to understad the collation failure you pointed out bellow.